packages/react/src/themes/teams/components/Popup/popupContentVariables.ts

```diff
@@ -13,9 +16,12 @@ export interface PopupContentVariables {
 
 export default (siteVars: any): PopupContentVariables => {
   return {
-    borderColor: siteVars.colors.grey[250],
+    borderColor: siteVars.colorScheme.default.border2,
     borderRadius: pxToRem(3),
```

Comment:
Should this be an explicit value instead of rems because of the border being 1px always?